core/templates/domain/creator_dashboard/creator-dashboard-backend-api.service.ts

```diff
@@ -85,7 +86,8 @@ interface CreatorDashboardData {
   suggestionThreadsToReviewList: SuggestionThread[];
   explorationsList: CreatorExplorationSummary[];
   collectionsList: CollectionSummary[];
-  topicSummaries: CreatorTopicSummary[];
+  // 'topicSummaryDicts' is null if no summary is provided for a topic.
+  topicSummaries: CreatorTopicSummary[] | null;
```

Comment:
Service is used to retrieve creator dashboard information from the backend. The field is present in backend as well
https://github.com/oppia/oppia/blob/db60b6e16226123d54a3529314d528051b102979/core/controllers/creator_dashboard.py#L151-L152
Will be null if summary is not present

Reply:
I see, but from the backend code it won't be null but an empty list, right?

Reply:
Converted to return an empty list instead of null here https://github.com/oppia/oppia/blob/26b1b5dad4b42a540b2557aba1d27b7bd099f190/core/templates/domain/creator_dashboard/creator-dashboard-backend-api.service.ts#L176-L179